ringpop.go

```diff
@@ -89,8 +89,7 @@ type Ringpop struct {
 		sync.RWMutex
 	}
 
-	logger log.Logger
-	log    log.Logger
+	moduleLogger *util.ModuleLogger
```

Comment:
Is the type enough of a hint of this? I'd prefer to keep the name simpler, if possible.